go/pkg/credshelper/credshelper.go

```diff
@@ -110,8 +112,8 @@ func buildExternalCredentials(baseCreds cachedCredentials, credsFile string, cre
 		TokenSource: oauth2.ReuseTokenSourceWithExpiry(
 			baseCreds.token,
 			baseTS,
-			// Refresh tokens 5 mins early to be safe
-			5*time.Minute,
+			// Refresh tokens a bit early to be safe
```

Comment:
super nit: this comment seems redundant :)

Reply:
In the next PR :)